### PR TITLE
#46046 Add tint to highlighted non-visible notes

### PR DESCRIPTION
**Pull request by TheRealFalcon**
For addressing https://musescore.org/en/node/46046

What it looks like:
screenshot_015

This is my first contribution to this project, so apologies if I missed a step or some due diligence.

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Please change the commit message to read "fix #46046: ..." so that the issue gets closed automagically once this PR got merged.
Make sure to have signed the [CLA](//musescore.org/cla)

File: libmscore/element.cpp [hunk 1]
```diff
@@ -304,10 +304,19 @@ QColor Element::curColor(const Element* proxy) const
             marked = note->mark();
             }
       if (proxy->selected() || marked ) {
+            QColor original_color;
```
Comment (Jojo-Schmitz) [inline libmscore/element.cpp]: In general we prefer camelCase, so better change this to `orginalColor`

**Comment by TheRealFalcon** [comment]
Updated based on comments. Is there a linter file for the project anywhere that I can run to check for things like code conventions?

**Comment by Jojo-Schmitz** [comment]
Yes, the linters are called laconic and Jojo-Schmitz ;-)

**Comment by Jojo-Schmitz** [comment]
Do I see it correctly that this works for all elements,  not just noteheads?

**Comment by TheRealFalcon** [comment]
It should apply to all elements that get highlighted within a staff. Clefs, key/time signatures, accidentals, articulations. 
screenshot_017

It doesn't apply to the dot, stem, or flag of a note, but that's because those don't get highlighted when we highlight a note. It also doesn't apply to things in the "Lines" category, texts, tempi, dynamics, or repeats. Tbh, I didn't think of things outside of the staff until you asked, so if we want things to be consistent, I can hunt those down too.

**Comment by Jojo-Schmitz** [comment]
(augmentation) dots stems and flags do get highlighted the same as noteheads accidentals etc., when selecting a measure

**Comment by TheRealFalcon** [comment]
Added another commit with the other color changes.

Screenshot with select all:
screenshot_019

Selecting some of the other things:
screenshot_020

Everything in these screenshots are non-visible items.
The repeat signs seem a bit off with the dots, but I see the same behavior on master when I select a repeat.

**Comment by Jojo-Schmitz** [comment]
I like it. Maybe the colors are a bit too faint though.

**Comment by TheRealFalcon** [comment]
Original:
point75

slightly darker:
point_6

much darker:
point5

I could go darker still, but I think that'd be too indistinguishable. What's your preference?

**Comment by Jojo-Schmitz** [comment]
The slightly darker looks good, can't we find our and reuse the same setting as for the 'invisible' grey?

**Comment by TheRealFalcon** [comment]
Alrighty, updated so the colors a little darker.

The non-visible grey setting is just `Qt::gray`, so I'm not sure the exact way to apply that same transformation to colored notes.

**Comment by Jojo-Schmitz** [comment]
Ah, yes, thanks for checking anyway

**Comment by lasconic** [comment]
Thanks! I merge it manually.
I used QColor::brighter() instead of your tint logic.

**Comment by TheRealFalcon** [comment]
:/

I tried brighter and didn't like the results. We want lighter, not brighter. Look at the green and the purple. They don't look "invisible", they look highlighted somehow, which is the opposite of what we want.


screenshot_004

**Comment by lasconic** [comment]
Mmm, ok. In fact I did use QColor::lighter and assumed it would do the same that what you did. Let me have a second look.

**Comment by lasconic** [comment]
Fixed. Thank you.

**Comment by TheRealFalcon** [comment]
Thanks!

**Pull request closed without merging**